src/background/hypothesis-chrome-extension.js

```diff
@@ -210,12 +225,23 @@ export default function HypothesisChromeExtension({
     });
   }
 
+  /**
+   *
+   * @param {chrome.tabs.Tab} tab
+   */
   function onTabCreated(tab) {
     // Clear the state in case there is old, conflicting data in storage.
-    state.clearTab(tab.id);
+    if (tab.id) {
```

Comment:
When might `tab.id` be nullish? Will there definitely still be a `tab` argument even if it has no ID?

Reply:
From the documentation:
`Tab ID can also be set to chrome.tabs.TAB_ID_NONE for apps and devtools windows.`